### ISSUE TITLE
[Suggestion]: code style improvement

### ISSUE THREAD

**Issue by Bearnardd** [anarchy-ai/LLM-VM#310]
Hi there!

I've recently delved into the codebase, and I've observed that there is room for style improvements. Currently, there is a considerable inconsistency in style both within individual files and between them.

Here's what I'm proposing:

(1) Review: I'll thoroughly review the codebase file by file and pinpoint areas where code style can be enhanced.

(2) Pull Requests (PRs): For each file that requires changes, I'll initiate a separate PR. This approach allows for a systematic review and discussion about the proposed changes.

**Comment by cartazio** [comment]
totes.

i'd suggest starting with the internal agents code because its desparate for love, and theres little activity there to coord around

**Comment by mmirman** [comment]
Yeah, we need review guidelines!  @us was really good at reviews, so maybe he can suggest a guideline we can use?

**Comment by mmirman** [comment]
I'm not the biggest fan of style guides though this early.

**Comment by daredevilteja** [comment]
I want to fix this issue. Is anyone working on this?

**Comment by daredevilteja** [comment]
This issue is currently on hold and will be addressed by the core team in future
![image](https://github.com/anarchy-ai/LLM-VM/assets/19908328/a5d36b8b-f3bc-483d-83f9-3ee80d91ba54)
